src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -300,13 +289,13 @@ const styles = StyleSheet.create({
     position: 'absolute',
     opacity: opacityHeaderAnimation
   }),
-  image: {
+  image: (followType) => ({
     width: normalize(48),
     height: normalize(48),
     borderRadius: normalize(24),
-    backgroundColor: 'lightgrey',
+    backgroundColor: followType === 'signed' ? 'lightgrey' : COLORS.anon_secondary,
     marginRight: 8
-  },
+  }),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The dynamic style assignment based on `followType` is a good way to apply different styles conditionally. Just ensure that the `styles.image` function handles all possible values of `followType`.


<!-- This is an auto-generated comment by OSS CodeRabbit -->